live/tests/integration/components/signin-form-test.js

```diff
@@ -2,88 +2,92 @@ import { expect } from 'chai';
+      expect(document.querySelector('input#pix-email')).to.exist;
     });
 
-    // Given
-    this.render(hbs`{{signin-form onSubmit=(action 'onSubmitAction')}}`);
+    it('should display an input for password field', function() {
+      // when
+      this.render(hbs`{{signin-form}}`);
 
-    _fillSigninForm(this, expectedEmail, expectedPassword);
+      // then
+      expect(document.querySelector('input#pix-password')).to.exist;
+    });
 
-    // When
-    this.$('button[type=submit]').click();
-  });
+    it('should display a submit button to authenticate', function() {
+      // when
+      this.render(hbs`{{signin-form}}`);
 
-  it('should also use action on submit', function(done) {
-    // Expect
-    this.on('onSubmitAction', (email, password) => {
-      expect(email).to.equal(expectedEmail);
-      expect(password).to.equal(expectedPassword);
-      done();
-      return Promise.resolve();
+      // then
+      expect(document.querySelector('button.signin-form__submit_button')).to.exist;
     });
 
-    this.render(hbs`{{signin-form onSubmit=(action 'onSubmitAction')}}`);
-    _fillSigninForm(this, expectedEmail, expectedPassword);
+    it('should display a link to password reset view', function() {
+      // when
+      this.render(hbs`{{signin-form}}`);
 
-    // When
-    this.$('.signin-form__form form').submit();
-  });
+      // then
+      expect(document.querySelector('a.signin-form__forgotten-password-link')).to.exist;
+    });
+
+    it('should not display any error by default', function() {
+      // when
+      this.render(hbs`{{signin-form}}`);
 
-  it('should display an error', function() {
-    // Expect
-    this.on('onSubmitAction', () => {
-      return Promise.resolve();
+      // then
+      expect(document.querySelector('div.signin-form__errors')).to.not.exist;
     });
 
-    this.render(hbs`{{signin-form onSubmit=(action 'onSubmitAction')}}`);
-    _fillSigninForm(this, expectedEmail, expectedPassword);
+    it('should display an error if authentication failed', function() {
+      // given
+      this.set('displayErrorMessage', true);
 
-    // When
-    this.$('.signin-form__form form').submit();
+      // when
+      this.render(hbs`{{signin-form displayErrorMessage=displayErrorMessage}}`);
 
-    // Then
-    expect(this.$('.signin-form__errors')).to.have.lengthOf(0);
+      // then
+      expect(document.querySelector('div.signin-form__errors')).to.exist;
+    });
   });
 
-  it('should hide the error message if it was previously displayed', function() {
-    // Expect
-    this.on('onSubmitAction', () => {
-      return Promise.resolve();
-    });
-    this.render(hbs`{{signin-form onSubmit=(action 'onSubmitAction') displayErrorMessage='true'}}`);
+  describe('Behaviours', function() {
 
-    expect(this.$('.signin-form__errors')).to.have.lengthOf(1);
-    _fillSigninForm(this, expectedEmail, expectedPassword);
+    it('should authenticate user when she submitted sign-in form', function() {
```

Comment:
when she ? :P

Reply:
En anglais inclusif, une des façons de régler le problème du genre, est de mixer he/she.

cf. http://www.learnersdictionary.com/qa/the-he-or-she-dilemma

> And some people have decided to get around the problem by using he in some sentences and she in others ("The student will need to show his ID to use the gym.  The student can rent a locker to store her clothes while she works out.").

Reply:
Autre source : https://blogs.mediapart.fr/michel-delarche/blog/141017/lecriture-inclusive-une-modeste-contribution-comparative